numpy/distutils/fcompiler/nv.py

```diff
@@ -0,0 +1,48 @@
+# https://developer.nvidia.com/hpc-sdk
+import sys
+
+from numpy.distutils.fcompiler import FCompiler
+
+compilers = ['NVHPCFCompiler']
+
+class NVHPCFCompiler(FCompiler):
```

Comment:
If you have any knowledge of how the nvidia compiler and pgfortran are related (or a link), a comment here containing that would likely be useful to someone in future.